### PR TITLE
IDEX-3955:Add handler for catching event then content inject to Editor Widget

### PR DESCRIPTION
**Pull request by vparfonov**
@evidolob @azatsarynnyy

### CODE REVIEW

File: ide/che-core-ide-jseditor/src/main/java/org/eclipse/che/ide/jseditor/client/texteditor/ContentInitializationHandler.java [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+ *******************************************************************************/
+package org.eclipse.che.ide.jseditor.client.texteditor;
+
+/**
+ * Handle event then conetnet will be injected to the Editor Widget
+ *
+ * @author Vitalii Parfonov
+ */
+public interface ContentInitializationHandler {
+
```
Comment (vitaliy-guliy) [inline ide/che-core-ide-jseditor/src/main/java/org/eclipse/che/ide/jseditor/client/texteditor/ContentInitializationHandler.java]: Maybe ContentInitializedHandler ?

File: ide/che-core-ide-jseditor/src/main/java/org/eclipse/che/ide/jseditor/client/texteditor/ContentInitializationHandler.java [hunk 2]
```diff
@@ -0,0 +1,21 @@
...
+    void onContentInitialized();
+}
```
Comment (vitaliy-guliy) [inline ide/che-core-ide-jseditor/src/main/java/org/eclipse/che/ide/jseditor/client/texteditor/ContentInitializationHandler.java]: onConrentInitialized(ContentInitializedEvent event)

File: ide/che-core-ide-jseditor/src/main/java/org/eclipse/che/ide/jseditor/client/texteditor/ContentInitializationHandler.java (hunk 1 above)
Comment (vparfonov) [inline ide/che-core-ide-jseditor/src/main/java/org/eclipse/che/ide/jseditor/client/texteditor/ContentInitializationHandler.java]: yes,thx

File: ide/che-core-ide-jseditor/src/main/java/org/eclipse/che/ide/jseditor/client/texteditor/ContentInitializationHandler.java [hunk 3]
```diff
@@ -0,0 +1,21 @@
...
+ * which accompanies this distribution, and is available at
+ * http://www.eclipse.org/legal/epl-v10.html
+ * <p>
+ * Contributors:
+ * Codenvy, S.A. - initial API and implementation
+ *******************************************************************************/
+package org.eclipse.che.ide.jseditor.client.texteditor;
+
+/**
+ * Handle event then conetnet will be injected to the Editor Widget
```
Comment (azatsarynnyy) [inline ide/che-core-ide-jseditor/src/main/java/org/eclipse/che/ide/jseditor/client/texteditor/ContentInitializationHandler.java]: Handle event **that**
no?

File: ide/che-core-ide-jseditor/src/main/java/org/eclipse/che/ide/jseditor/client/texteditor/EditorWidget.java [hunk 4]
```diff
@@ -61,8 +64,9 @@
      *
      * @param newValue
      *         the new contents
+     * @param initializationHandler must be called then content injected  the Editor Widget
```
Comment (azatsarynnyy) [inline ide/che-core-ide-jseditor/src/main/java/org/eclipse/che/ide/jseditor/client/texteditor/EditorWidget.java]: must be called **when** content injected **in** the Editor Widget

File: ide/che-core-ide-jseditor/src/main/java/org/eclipse/che/ide/jseditor/client/texteditor/ContentInitializationHandler.java (hunk 2 above)
Comment (vzhukovskii) [inline ide/che-core-ide-jseditor/src/main/java/org/eclipse/che/ide/jseditor/client/texteditor/ContentInitializationHandler.java]: It's simple handler interface, not gwt event handler

File: ide/che-core-ide-jseditor/src/main/java/org/eclipse/che/ide/jseditor/client/texteditor/ContentInitializationHandler.java (hunk 1 above)
Comment (azatsarynnyy) [inline ide/che-core-ide-jseditor/src/main/java/org/eclipse/che/ide/jseditor/client/texteditor/ContentInitializationHandler.java]: +1

File: ide/che-core-ide-jseditor/src/main/java/org/eclipse/che/ide/jseditor/client/texteditor/EditorWidget.java (hunk 4 above)
Comment (vparfonov) [inline ide/che-core-ide-jseditor/src/main/java/org/eclipse/che/ide/jseditor/client/texteditor/EditorWidget.java]: yes

File: ide/che-core-ide-jseditor/src/main/java/org/eclipse/che/ide/jseditor/client/texteditor/ContentInitializationHandler.java (hunk 3 above)
Comment (sleshchenko) [inline ide/che-core-ide-jseditor/src/main/java/org/eclipse/che/ide/jseditor/client/texteditor/ContentInitializationHandler.java]: Also take a look *conetnet*

**Pull request merged**